### PR TITLE
feat: krystal earn

### PR DESCRIPTION
**Pull request by phucledien**
**What does this PR do?**

-   [x] migrate krystal-api -> mochi-api for fetching invest list
-   [x] Update `/invest` to `/invest info` 
-   [x] Stake
-   [x] View invest request status
-   [x] View invest portfolio

**How to test**

**Media (Loom or gif)**

CleanShot 2023-07-17 at 09 22 51

### CODE REVIEW

File: src/commands/invest/stake/slash.ts [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+          )}% - TVL: ${formatter.format(p.tvl ?? 0)}`,
+          value: p.name ?? "NA",
+        }))
+
+      await i.respond(options ?? [])
+    }
+  },
+  run: async function (i: CommandInteraction) {
+    const chainId = i.options.getString("chain", true)
+    const [, tokenAddress] = i.options.getString("token", true).split("|")
```
Comment (tuanddd) [inline src/commands/invest/stake/slash.ts:129]: split bằng space thì intuitive hơn chứ hả

File: src/commands/invest/stake/slash.ts [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+        }))
+
+      await i.respond(options)
+    }
+
+    if (focused.name === "platform") {
+      const focusedValue = focused.value
+      const chainId = i.options.getString("chain", true)
+      const token = i.options.getString("token", true)
+      const address = token.split("|")[1]
```
Comment (tuanddd) [inline src/commands/invest/stake/slash.ts:95]: split bằng space sẽ ok hơn?

Comment (phucledien) [reply src/commands/invest/stake/slash.ts:95]: ok ser

File: src/commands/invest/stake/slash.ts (hunk 1 above)
Comment (phucledien) [reply src/commands/invest/stake/slash.ts:129]: ok ser

**Pull request merged**